project/Dependencies.scala

```diff
@@ -19,7 +19,7 @@ object Dependencies {
 
     // If you update the version of Play, you probably need to update the other Play* variables.
     val Play = "2.7.0"
-    val PlayJson = "2.7.0"
+    val PlayJson = "2.7.2"
```

Comment:
Btw, we are about to release Play 2.7.1. Maybe we should hold this PR and update all Play related deps in one shot.

Reply:
You don't need to close. We can update this PR when the moment arrives.